bin/templatemap_generator.php

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/env php
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
```

Comment:
I think @Ocramius comment is about placing license header, not about where template map generator